views/pages/AccountPage/AccountBanner.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
I think this file name should be `AccountPageHeader.test.jsx`

Reply:
[a8fa89f](https://github.com/AMP-SCZ/dpdash/pull/478/commits/a8fa89f3d10b6e4db122a9aa52573c428d3682dc)